src/utils/store.js

```diff
@@ -1,8 +1,149 @@
-const store = require('store2');
-const overflow = require('./store.overflow');
+const session = {
+  overflow: {},
+  get(key) {
+    // Overflow
+    const valueFromOverflow = this.overflow[key];
 
-// Inject overflow into store
-// https://raw.githubusercontent.com/nbubna/store/master/src/store.overflow.js
-overflow(store, store._);
+    if (valueFromOverflow) {
+      return valueFromOverflow;
+    }
+
+    if (typeof sessionStorage === 'undefined') {
+      return null;
+    }
+
+    const valueFromSession = sessionStorage.getItem(key);
+
+    if (valueFromSession) {
+      return JSON.parse(valueFromSession);
```

Comment:
If `valueFromSession`is invalid JSON that couldn't be parsed this would through an error that's not handled right? If so can we wrap this in a try catch to handle exception